### ISSUE TITLE
Note text box incorrectly changes appearance in certain cases

### ISSUE THREAD

**Issue by clong1995** [spheras/desktopfolder#67]
是这样的，我遇到一个bug，新建的note，中间文字部分的背景是透明的，但是文字四周是有背景颜色的。
系统版本 elementary OS 0.4.1 Loki 64bit 
显卡intel和nvidia

---

*Edit by aljelly (Google Translate)*:

This is the case, I encountered a bug, the new note, the middle of the text part of the background is transparent, but the text around the background color.
System version elementary OS 0.4.1 Loki 64bit
Graphics intel and nvidia

**Comment by aljelly** [comment]
I think I've noticed what you're describing. I've noticed it while using Desktop Folder in a virtual machine. It hasn't happened when elementary's been installed on my PC (using nouveau).

If the issue you're describing is what I believe it is: when it happens, the appearance of the text box inside the note changes to a color/transparency that doesn't fit with the surrounding note background color. I think it might be when trying to edit the text inside it.

I'm not exactly sure what would be causing it. It could be to do with GtkSourceView and the note window being transparent.

Does it look anything like this?

![screenshot from 2017-11-06 12 23 47](https://user-images.githubusercontent.com/10395308/32440989-6b6e5ee2-c2ed-11e7-9686-4c145beb434f.png)

The window was modified to look like this. This is does not represent what the bug actually looks like, it's an approximation of what I recall it looking like.

(This issue needs to be confirmed.)
